test/end-to-end/pages/screeners/applicantexecutor.js

```diff
@@ -11,8 +11,8 @@ module.exports = async function(language ='en', answer) {
     const applicantExecutorContent = language === 'en' ? applicantExecutorEn : applicantExecutorCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/applicantexecutor');
+    await I.waitForText(applicantExecutorContent.question);
     if (language === 'en') {
```

Comment:
could we not have a welsh check here also

Reply:
done.